LibreHardwareMonitor/Utilities/HttpServer.cs

```diff
@@ -563,16 +569,106 @@ private async Task SendJsonAsync(HttpListenerResponse response, HttpListenerRequ
         response.Close();
     }
 
+    private string SendPrometheusTreeTransversal(Node node)
```

Comment:
A better name would perhaps be ```GetPrometheusResponse```

Reply:
I was trying to group a "TreeTransversal" under the "SendPrometheus" function. That funciton is not really Getting a response from prometheus, how about `GeneratePrometheusResponse` ? Because it is Generating a response in a prometheus format.